lib/dataElementListener.js

```diff
@@ -60,7 +60,14 @@ class DataElementImporter extends SHRDataElementParserListener {
 
   enterDoc(ctx) {
     // Process the namespace
-    const ns = ctx.docHeader().namespace().getText();
+    let ns = '';
+    if (ctx.docHeader().namespace()) {
+      ns = ctx.docHeader().namespace().getText();
+    }
+    else {
+      logger.error('Namespace not present in file. ERROR_CODE:11???');
```

Comment:
Can we change the error message to `'Namespace declaration not found. ERROR_CODE:11???'`